### PR TITLE
doc(readme): Add information about PostgreSQL password

### PR DESCRIPTION
**Pull request by TheFaR8**
### Problem

When I'm building BookBrainz from scratch, I'm having a difficulty with PostgreSQL Password

### Solution

In order to set things right and not to confuse others who want to build BookBrainz, I added information about PostgreSQL password

### Areas of Impact

Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,19 @@ earlier):
 
     sudo -u postgres psql bookbrainz -c "SELECT name FROM bookbrainz.editor"
 
+You are also required to set the password of PostgreSQL for your local.
+you can do this by
+
+
+    sudo -u postgres psql
+    psql (9.6.9)
+    Type "help" for help.
```
Comment (MonkeyDo) [inline README.md]: I would remove the two lines `psql (9.6.9)` and `Type "help" for help.` to limit possible confusion.

File: README.md [hunk 2]
```diff
@@ -83,6 +83,19 @@ earlier):
 
     sudo -u postgres psql bookbrainz -c "SELECT name FROM bookbrainz.editor"
 
+You are also required to set the password of PostgreSQL for your local.
```
Comment (MonkeyDo) [inline README.md]: How about rephrasing it as such: "You are also required to set the password of your local PostgreSQL instance." ?

**Comment by TheFaR8** [comment]
Done :)

**Comment by MonkeyDo** [comment]
Thanks @TheFaR8 !

**Comment by TheFaR8** [comment]
You're welcome :)

**Pull request merged**